plugins/herald.py

```diff
@@ -45,6 +45,19 @@ def herald(text, nick, chan, db, conn):
         db.commit()
         return("greeting successfully added")
 
+@hook.command()
```

Comment:
There should be permissions set on the command. Changing line 48 to `@hook.command(permissions=["botcontrol", "snoonetstaff"])` should be all you need to do for that.